owiddata/owiddata-stats.json

```diff
@@ -1,28 +1,24 @@
 {
-  "owid_RNA_count": 3,
-  "owid_RNA_countries": 167,
-  "owid_RNA_map": "https://github.com/greenelab/covid19-review/raw/5eeb401b6b2cf424cfaf47e86708f9c873bd6242/owiddata/maps/RNA.png",
-  "owid_VLP_count": 1,
+  "owid_RNA_count": 2,
```

Comment:
The old values were from VIPER rather than OWID. This is corrected so that the variable name matches the data source. I plan to add the VIPER values in in a new PR.